customize_image.sh

```diff
@@ -314,27 +314,20 @@ if [ -e "$IMAGEDIR/etc/init.d/apache2" ] ; then
 fi
 
 if [ "$MINIMG" != "1" ]; then
-	# fake a revpi procfs ... otherwise NodeRed installer will get confused
-	for procfile in meminfo cpuinfo; do
-		touch "$IMAGEDIR/proc/$procfile"
-		mount -o ro,bind "$BAKERYDIR/templates/fake-procfs/$procfile" "$IMAGEDIR/proc/$procfile"
-	done
-
-	# install nodejs and nodered with an install script and revpi-nodes from npm repository
-	NODEREDSCRIPT="/tmp/update-nodejs-and-nodered.sh"
-	/usr/bin/curl -sL \
-		https://raw.githubusercontent.com/node-red/linux-installers/master/deb/update-nodejs-and-nodered\
-		--output "$IMAGEDIR/$NODEREDSCRIPT"
-	chmod 755 "$IMAGEDIR/$NODEREDSCRIPT"
+	# Use NodeJS LTS 18 sources
+	cp "$BAKERYDIR/templates/nodered/nodesource.gpg" "$IMAGEDIR/etc/apt/trusted.gpg.d"
+	cp "$BAKERYDIR/templates/nodered/nodesource.list" "$IMAGEDIR/etc/apt/sources.list.d"
+
+	# Install nodejs 18 including npm
+	chroot "$IMAGEDIR" apt-get update
+	chroot "$IMAGEDIR" apt-get -y install nodejs
```

Comment:
Can we specify a specific version to install with `apt`? The comment would get outdated for the next image we decide to build and I'm afraid we might miss it. This would make building the image more reproducible.

Something like a global var for the `nodejs` package version?

Reply:
As soon as we tanke the nodejs 18 sources it will be always the version 18-LTS. And whether we install the latest one or the customer installs a newer one through an update does not matter at all.

Reply:
I forgot we give the source for nodejs 18 in `nodesource.list`.

I'd still prefer the comment to be independent of the version, though. Maybe something like:

```suggestion
	# Install the nodejs version from `nodesource.list` added above, including npm
	chroot "$IMAGEDIR" apt-get update
	chroot "$IMAGEDIR" apt-get -y install nodejs
```

This way we avoid that we forget to update the comment if we decide to update the nodejs version in `nodesource.list`.